src/Data/Dwarf.hs

```diff
@@ -95,7 +105,10 @@ import Data.Dwarf.Reader
 import Data.Dwarf.TAG
 import Data.Dwarf.Types
 import qualified Data.Map.Strict as M
-import Data.Word (Word64)
+import Data.Word (Word64, Word8)
+import Data.Maybe (fromMaybe)
+import Control.Monad.Trans.State (StateT (runStateT), get, put, evalStateT)
+import Control.Monad.Trans.Class (lift)
```

Comment:
This module is adhering to the convention of alphabetizing its imports, so let's stick that convention with the new `Control.Monad.Trans.*` imports.